docs/governance/overview.md

```diff
@@ -1,30 +1,25 @@
 
----
+# On-chain Voting
 
-Decred’s governance is based on the principle of ticket-holder voting. To participate in governance, users must time-lock DCR to purchase [tickets](../proof-of-stake/overview.md). Tickets can then be used to vote on validating blocks, consensus rule changes, or proposals in Decred’s web-based proposal system, [Politeia](https://proposals.decred.org/). 
+On-chain voting serves two main purposes: validating blocks created by Proof-of-Work (PoW) miners and voting on proposed consensus rule changes.
 
-Validation of blocks and voting on consensus rule changes occurs on-chain. Voting on higher-level issues, such as how to spend [Treasury funds](http://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx), or significant policy decisions (e.g. amend the [Decred constitution](decred-constitution.md)) occurs off-chain in Politeia, a web platform where the community at large can submit, discuss and vote on proposals.
+## Block Voting
 
-To compensate ticket holders for validating blocks and securing the network, 30% of the block reward (PoS reward) is sent to ticket holders for voting on blocks; the remaining block reward is split between PoW miners (60%) and the Treasury (10%) to fund ongoing development. The ticket price (the amount of DCR that must be time-locked) is adjusted dynamically every 144 blocks (~12 hrs), in an attempt to keep PoS returns stable over time. 
+In each block, five tickets are selected pseudo-randomly and called to [vote on-chain](../proof-of-stake/overview.md). Tickets then vote to approve or reject the previous block of transactions created by a PoW miner. At least three out of five tickets called to vote must respond for the block to be considered valid. This power provides a check against malicious PoW miner behavior (e.g. mining empty blocks), as well as forming the basis of Decred’s fork resistance. After a ticket is called to vote, the DCR locked in that ticket is unlocked and returned to the purchaser’s wallet, along with the PoS reward if the ticket successfully voted. Decred is a hybrid PoW/PoS system.
 
+## Consensus Rule Voting
 
----
+Tickets can also vote about [consensus rule changes](consensus-rule-voting/overview.md). A proposed change must be approved by 75% of non-abstaining tickets to take effect. This way a fork is less likely to occur. The change will automatically activate, so this is a change in the software itself, and not just theoretical.
 
-## On-chain voting
+# Off-chain Voting
 
-On-chain voting serves two main purposes: validating blocks created by Proof-of-Work (PoW) miners and voting on proposed consensus rule changes. 
+Decisions concerning the direction of the project, such as how to spend Treasury funds, or amending the Decred Constitution or other policies, are made via proposals on Politeia.
```

Comment:
Link to the constitution here because the other link has been removed

Reply:
Also move the link to politeia to this line, because this is the first time politeia is mentioned